ansible/roles/cas5/templates/well-known

```diff
@@ -0,0 +1,186 @@
+{
+  "issuer": "{{auth_cas_url}}/oidc",
+  "scopes_supported": [
+{% for ascope in cas_oidc_well_known_scopes %}
+    "{{ ascope }}"{% if not loop.last %},{%endif%}
+    
+{% endfor %}
+  ],
+  "response_types_supported": [
+    "code",
+    "token",
+    "id_token token"
+  ],
+  "subject_types_supported": [
+    "public",
+    "pairwise"
+  ],
+  "claim_types_supported": [
+    "normal"
+  ],
+  "claims_supported": [
```

Comment:
This is fine but is technically derived from application config (the static app config that's built into the JAR).  Something to keep in mind if the app config is updated.